Makefile

```diff
@@ -79,16 +79,13 @@ binappend:
 $(CURDIR)/bin/crc$(IS_EXE):
 	go install -ldflags="$(VERSION_VARIABLES)" ./cmd/crc
 
-$(BUILD_DIR)/$(GOOS)-$(GOARCH):
-	mkdir -p $(BUILD_DIR)/$(GOOS)-$(GOARCH)
-
-$(BUILD_DIR)/darwin-amd64/crc: $(BUILD_DIR)/$(GOOS)-$(GOARCH)
+$(BUILD_DIR)/darwin-amd64/crc:
 	GOARCH=amd64 GOOS=darwin go build -ldflags="$(LDFLAGS)" -o $(BUILD_DIR)/darwin-amd64/crc ./cmd/crc
```

Comment:
This is not the intended for this commit change but can we also use env for running the `go build`?


```
# From
GOARCH=amd64 GOOS=darwin go build -ldflags="$(LDFLAGS)" -o $(BUILD_DIR)/darwin-amd64/crc ./cmd/crc

# To
GOARCH=amd64 GOOS=darwin go build -ldflags="$(LDFLAGS)" -o $(BUILD_DIR)/$(GOOS)-$(GOARCH)/crc ./cmd/crc
```

Reply:
Good idea, but I did not manage to get it to work :(